contrib/plugins/albumartist_website.py

```diff
@@ -77,10 +77,12 @@ def website_add_track(self, album, track, artistId):
         if self.website_queue.append(artistId, (track, album)):
             host = config.setting["server_host"]
             port = config.setting["server_port"]
-            path = "/ws/2/%s/%s?inc=%s" % ('artist', artistId, 'url-rels')
+            path = "/ws/2/%s/%s" % ('artist', artistId)
+            queryargs = {"inc": "url-rels"}
             return album.tagger.xmlws.get(host, port, path,
                         partial(self.website_process, artistId),
-                        xml=True, priority=True, important=False)
+                                xml=True, priority=True, important=False,
+                                queryargs=queryargs)
 
```

Comment:
Fixing it in contrib/plugins obviously doesn't need any forward / backward compatibility settings, however a fix which provides backwards compatibility is needed in the separate picard_plugins repo.